aisynphys/database/schema/experiment.py

```diff
@@ -195,7 +195,8 @@ def __repr__(self):
         ('experiment_id', 'experiment.id', '', {'index': True}),
         ('pre_cell_id', 'cell.id', 'ID of the presynaptic cell', {'index': True}),
         ('post_cell_id', 'cell.id', 'ID of the postsynaptic cell', {'index': True}),
-        ('has_synapse', 'bool', 'Whether a chemical synapse was manually detected for this cell pair', {'index': True}),
+        ('has_synapse', 'bool', 'Whether a chemical mono-synapse was manually detected for this cell pair', {'index': True}),
+        ('has_polysynapse', 'bool', 'Whether a chemical poly-synapse was manually detected for this cell pair', {'index': True}),
```

Comment:
If it's polysynaptic, then we no longer have a way to distinguish electrical from chemical synapses. Recommend: "whether a polysynaptic connection was manually detected for this cell"